docs/API/Elements.md

```diff
@@ -5,18 +5,60 @@ title: Elements
 
 The Elements API is a powerful abstraction that allows users to manipulate any property on any element through a handful of simple methods. This is important because these few methods allow for easy networking or persistent state. 
 
+To change these properties you will need to reference an object. The most common form to reference an object would be:
+
+```js
+var a = this; 
+```
```

Comment:
Don't forget to add a `<br>` here, in order to keep spacing between code consistent on the page